src/tox/config.py

```diff
@@ -413,6 +415,15 @@ def tox_addoption(parser):
         dest="sdistonly",
         help="only perform the sdist packaging activity.",
     )
+    parser.add_argument(
+        "--parallel", action="store_true", dest="parallel", help="run tox environments in parallel"
+    )
+    parser.add_argument(
+        "--parallel-live",
```

Comment:
Wouldn't it be better to use the verbosity level here? Something like:

* 0 no output from parallel run
* 1 simple progress bar (basically just showing that the whole thing is not stuck)
* 2+ life logs with growing verbosity (error -> warning -> info -> debug)